src/ai/backend/common/redis.py

```diff
@@ -258,8 +262,11 @@ async def execute(
                         return newdict
                 else:
                     return result
+        except aioredis.exceptions.ConnectionError as e:
+            log.exception(f'execute(): Connecting to redis failed: {e}')
+            await asyncio.sleep(reconnect_poll_interval)
+            continue
```

Comment:
Let's move this `except` block below the following block, as the following block catches more specific errors.
Fortunately the exception type hierarchy does not overlap, but it is always better to catch more specific errors earlier.